clients/src/main/java/org/apache/kafka/clients/Metadata.java

```diff
@@ -248,6 +250,13 @@ public synchronized long lastSuccessfulUpdate() {
     }
 
     /**
+     * The max allowable age of metadata.
+     */
+    public synchronized long getMetadataMaxAge() { 
```

Comment:
Minor: In Kafka we usually do not specify "get" in the getters. "getXXX" usually means the code will generate the return value after some kind of calculation. So in this case, the method name should probably by something like `metadataMaxAge()` or `maxAge()`. Also, it seems that we don't need to synchronize on this method.